sdk/src/main/java/com/mapbox/maps/StyleObserver.kt

```diff
@@ -122,26 +124,20 @@ internal class StyleObserver(
   }
 
   private fun onStyleSourcesReady() {
-    // sources listener is not null - it means style does not contain them thus StyleDataType.SOURCES
-    // was not fired, explicitly call it to bind all the necessary style extensions upstream
-    styleDataSourcesLoadedListener?.let {
+    styleDataSourcesLoadedListener?.let { listener ->
       // reset listener first, firing onStyleLoaded
       // may produce another StyleDataType.SOURCES event if sources are added
       styleDataSourcesLoadedListener = null
-      it.onStyleLoaded(
-        preLoadedStyle ?: throw MapboxMapException("Style is not initialized yet although SOURCES event has arrived!")
-      )
+      preLoadedStyle?.let(listener::onStyleLoaded)
+        ?: logE(TAG, "Style is not initialized yet although SOURCES event has arrived!")
     }
   }
 
   private fun onStyleSpritesReady() {
-    // sprites listener is not null - it means style does not contain them thus StyleDataType.SPRITE
-    // was not fired, explicitly call it to bind all the necessary style extensions upstream
```

Comment:
Are those comments not relevant anymore?

Reply:
They are moved to the place where they _are_ relevant =)